ui/apps/ui/src/app/components/widgets/videos.component.ts

```diff
@@ -5,7 +5,7 @@ import { environment } from '@environment/environment';
 import { Observable, map } from 'rxjs';
 import { HttpClient } from '@angular/common/http';
 import { VideoDetail } from '../../widgets/videos/videos-widget.types';
-const YOUTUBE_API_KEY = 'we need key here';
+const YOUTUBE_API_KEY = 'AIzaSyAlLHVx3PSPvVnCUd7DL2wNsj_9UzLPTAk';
 const YOUTUBE_API_URL = 'https://www.googleapis.com/youtube/v3/search';
```

Comment:
I think these two constants should be moved to environment.ts.

Reply:
yep, im moving those to backend in next PR